tests/magick-image/add-profile.spec.ts

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright Dirk Lemstra https://github.com/dlemstra/magick-wasm.
+// Licensed under the Apache License, Version 2.0.
+
+import { IMagickImage, MagickImage } from '../../src/magick-image';
+import { TestImages } from '../test-images';
+
+let image: IMagickImage;
+
+beforeEach(() => {
```

Comment:
I am trying to get rid of the `beforeEach` and `afterEach` blocks but I don;t think I have a solution yet for this specific situation. We will need to take a look at that after this is merged.